### PR TITLE
Calypso Build: add more dependencies that need to be transpiled

### PR DESCRIPTION
**Pull request by jeherve**
#### Changes proposed in this Pull Request

The Jetpack plugin relies on Calypso Build, and we've recently ran into some issues with some dependencies:

- Query String (https://github.com/sindresorhus/query-string/issues/189)
- Split on First (https://github.com/sindresorhus/split-on-first/issues/6)
- Strict URI Encode (https://github.com/kevva/strict-uri-encode/issues/6)

Those dependencies now use es6, and consequently need to be transpiled before they can be used.

Since those are fairly general modules, I think it would make sense to patch them in Calypso Build right away; other Calypso Build users may rely on the as well, directly or indirectly.

#### Testing instructions

1. Start with [a local installation of Jetpack](https://github.com/Automattic/jetpack/blob/master/docker/README.md)
2. Run `yarn build` to build the dependencies
3. Edit `node_modules/@automattic/calypso-build/webpack/util.js` with the changes from this PR.
4. Run `yarn build-extensions`
5. Visit your site's block editor in IE11; Jetpack blocks should be available.

### CODE REVIEW

**Review by bisko — APPROVED** [review]
The proposed changes do fix the issue in Jetpack's context.

The rest of the experience and look of the block in IE11 looks pretty bad, but that should be addressed in a separate PR:

Screenshot 2020-07-06 at 15 38 23

**Review by pento — APPROVED** [review]
This fix works for me.

That toolbar misalignment is a Gutenberg issue, see WordPress/gutenberg#21389.

**Comment by jeherve** [comment]
@Automattic/team-calypso I'd be happy to get your opinion and review on this. Thank you!

File: packages/calypso-build/webpack/util.js
Comment (scinos) [inline packages/calypso-build/webpack/util.js:93]: These two are missing the trailing `/`
```suggestion
	'split-on-first/',
	'strict-uri-encode/',
```

**Review by scinos — CHANGES_REQUESTED** [review]
LGTM, assuming the trailing `/` is added for consistency before merging it to master.

File: packages/calypso-build/webpack/util.js
Comment (jeherve) [reply packages/calypso-build/webpack/util.js]: Good point. Done in a27f05454ac12e19edc507bfe010ef06948157be

**Review by david-szabo97 — COMMENTED** [review]
I'd like to have this PR in the next version as well:
https://github.com/Automattic/wp-calypso/pull/45433

What do you think about leaving version as-is (6.2.0) then once this and #45433 is merged, then we create a PR to bump the version?

**Comment by jeherve** [comment]
That works for me. I've reverted my version changes, so we should be ready to merge with this one, and you can bump the version after your changes. 👍

**Comment by sirreal** [comment]
It would be great to make this happen in Jetpack to catch these build issues ahead of time: https://github.com/Automattic/jetpack/issues/13693

I think we package up that linter as a standalone `@automattic/…` package without much difficulty.

**Comment by simison** [comment]
@sirreal @david-szabo97 do we need to run that linter for editor kit plugin as well?

**Comment by david-szabo97** [comment]
@simison Yes! We had the same problem there: https://github.com/Automattic/wp-calypso/pull/45433

**Pull request merged**
